themes/default/content/docs/intro/concepts/resources/names.md

```diff
@@ -69,6 +69,13 @@ resources:
     type: aws:iam:Role
 ```
 
+{{% notes type="info" %}}
+
+Pulumi YAML does not distinguish between logical and lexical
```

Comment:
I take this as a side-note meaning "there's no variable names, you just use resource names". I think this is worth telling people; perhaps it could be in plainer language though? I am thinking not everyone will grok "lexical" (and perhaps I didn't ..). How about:

> In Pulumi YAML, you always use the logical name of a resource (e.g., `my-role` in the above) to refer to it. There are no variable names, as there are in other languages.